### PR TITLE
data: prathamkandari

### PR DESCRIPTION
**Pull request by prathamkandari**
## Changes proposed

Contributing to OpenSource 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5363"></a>

### CODE REVIEW

**Review by SurajPratap10 — APPROVED** [review]
LGTM

File: data/prathamkandari.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Pratham Kandari",
+  "type": "personal",
+  "bio": "**Cloud Enthusiast**",
+
+  "tags": ["WebDev", "DataScience", "OpenSource", "Cloud"],
+
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/<Your Twitter Username>" },
```
Comment (Pradumnasaraf) [inline data/prathamkandari.json:9]: Add your Twitter usernames, if not, remove the section.

File: data/prathamkandari.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    {
+      "name": "GitHub: Let's collaborate",
+      "url": "https://github.com/prathamkandari",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/<Your Twitter Username>",
+      "icon": "FaTwitter"
+    },
```
Comment (Pradumnasaraf) [inline data/prathamkandari.json:27]: Add your Twitter usernames, if not, remove the section.

```suggestion
    {
      "name": "Twitter: Follow me",
      "url": "https://twitter.com/<Your Twitter Username>",
      "icon": "FaTwitter"
    },
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

There are some missing elements in the JSON file please update them to make the PR ready for merge

**Comment by SaraJaoude** [comment]
@prathamkandari the changes that have been proposed by the maintainer above would enhance your LinkFree profile.  However as there has been a lack of activity from you to make these changes I will be merging your Pull Request.

Should you wish to make changes please raise a new Pull Request.  You may wish to look at our Editing Guide in Docs when you do so:https://linkfree.eddiehub.io/docs/how-to-guides/editing

**Pull request merged**